customer_outstanding_statement/views/statement.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="customer_outstanding_statement.statement_document">
+        <t t-call="web.external_layout">
+            <div class="page">
+                <div class="row">
+                    <div class="col-xs-5 col-xs-offset-7">
+                        <span t-field="o.name"/><br/>
+                        <span t-raw="o.contact_address.replace('\n\n', '\n').replace('\n', '&lt;br&gt;')"/>
+                        <span t-field="o.vat"/>
+                    </div>
+                    <h4 style="padding-left:20em">
+                        Outstanding Statement
+                    </h4>
+                    <p>
+                        Date: <span t-esc="Date[o.id]" /><br/>
```

Comment:
@hveficent Here is taken today, not the date_end from the wizard.

Reply:
Obviously, but this is correct. The report has to mark the date of when the report was printed.